model/src/main/scala/org/zalando/nakadi/client/model/Model.scala

```diff
@@ -0,0 +1,192 @@
+ * @param effectiveSchema The effective schema of this `EventType`. The predefined schema validator will use the value of this field as base for the validation. The creation of EventTypes of different categories implies a distinct general structure for the payload of events of each type. This ultimate format the payload must adhere to is reflected in the `effective_schema`. See description of field `category` for more details of its generation. This field is informative and its contents cannot be manipulated directly.
+ * @param validationStrategies Determines the validation that has to be executed upon reception of Events of this type. Events are rejected if any of the rules fail (see details of Problem response on the Event publishing methods). Rule evaluation order is the same as in this array. If not explicitly set, default value will respect the definition of the `EventType.category`.
+ * @param enrichmentStrategies Determines the enrichment to be performed on an Event upon reception. Enrichment is performed once upon reception (and after validation) of an Event and is only possible on fields that are not defined on the incoming Event. See documentation for the write operation for details on behaviour in case of unsuccessful enrichment.
+ * @param partitionResolutionStrategy Determines how the assignment of the event to a Partition should be handled.
+ * @param schema The schema for this EventType. This is expected to be a json schema in yaml format (other formats might be added in the future).
+ * @param dataKeyFields Indicators of the path of the properties that constitute the primary key (identifier) of the data within this Event. If set MUST be a valid required field as defined in the schema. (TBD should be required? Is applicable only to both Business and DataChange Events?)
+ * @param partitioningKeyFields Indicator of the field used for guaranteeing the ordering of Events of this type (used by the PartitionResolutionStrategy). If set MUST be a valid required field as defined in the schema.
+ *
+ */
+case class EventType(
+  name: String,
+  owner: String,
+  category: String,
+  effectiveSchema: String,
+  validationStrategies: Seq[String],
+  enrichmentStrategies: Seq[String],
+  partitionResolutionStrategy: PartitionResolutionStrategy, //Different naming
+  schema: EventTypeSchema,
+  dataKeyFields: Seq[String],
+  partitioningKeyFields: Seq[String])
+
+/**
+ * The schema for an EventType, expected to be a json schema in yaml format (other formats might be added in the future).
+ * @param type The type of schema definition (avro, json-schema, etc).
+ * @param schema The schema as string in the syntax defined in the field type. Failure to respect the syntax will fail any operation on an EventType.
+ */
+case class EventTypeSchema(
+  schemaType: String,
+  schema: String)
+
```

Comment:
EventTypeStatistics is missing:
https://github.com/zalando/nakadi/blob/nakadi-jvm/api/nakadi-event-bus-api.yaml#L1234